docs/eventlisteners.md

```diff
@@ -27,47 +92,60 @@ rules:
   verbs: ["create"]
 ```
 
-Note that currently, JSON is the only accepted MIME type for events.
+If your EventListener is using
+[`ClusterTriggerBindings`](./clustertriggerbindings.md), you'll need a
+ServiceAccount with a
+[ClusterRole instead](../examples/role-resources/clustertriggerbinding-roles/clusterrole.yaml)
+
+### ServiceType
+
+The `serviceType` field is optional. EventListener sinks are exposed via
+[Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types).
+By default, the serviceType is `ClusterIP` which means any pods running in the
+same Kubernetes cluster can access services' via their cluster DNS. Other valid
+values are `NodePort` and `LoadBalancer`. Check the
+[Kubernetes Service types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
+documentations for details.
+
+For external services to connect to your cluster (e.g. GitHub sending webhooks),
+check out the guide on [exposing EventListeners](./exposing-eventlisteners.md)
```

Comment:
missing a period `check out the guide on [exposing EventListeners](./exposing-eventlisteners.md).`

Reply:
done